File: 1. Beginner/3241/3241.cs
using System;

class URI {
    static void Main() {
        int n = int.Parse(Console.ReadLine());

        for (int i = 0; i < n; i++) {
            string s = Console.ReadLine();
            if (s == "P=NP") {
                Console.WriteLine("skipped");
            } else {
                string[] parts = s.Split('+');
                int a = int.Parse(parts[0]);
                int b = int.Parse(parts[1]);
                Console.WriteLine(a + b);
            }
        }
    }
}

File: 1. Beginner/3241/3241.dart
import 'dart:io';

void main() {
    int n = int.parse(stdin.readLineSync()!);

    for (int i = 0; i < n; i++) {
        String s = stdin.readLineSync()!;
        if (s == 'P=NP') {
            print('skipped');
        } else {
            List<String> parts = s.split('+');
            int a = int.parse(parts[0]);
            int b = int.parse(parts[1]);
            print(a + b);
        }
    }
}